### PR TITLE
Create getProposedFederationCreationTime Bridge Method

### PR DESCRIPTION
**Pull request by apancorb**
## Description
Implements the Bridge method `getProposedFederationCreationTime`

## Motivation and Context
https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP419.md

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/BridgeTest.java [hunk 1]
```diff
@@ -2266,6 +2266,40 @@ void getProposedFederationAddress(MessageCall.MsgType msgType, ActivationConfig
...
+            .transaction(rskTxMock)
+            .activationConfig(activationConfig)
+            .bridgeSupport(bridgeSupportMock)
+            .msgType(msgType)
+            .build();
+
+        CallTransaction.Function function = BridgeMethods.GET_PROPOSED_FEDERATION_CREATION_TIME.getFunction();
+        byte[] data = function.encode();
+
+        if (activationConfig.isActive(ConsensusRule.RSKIP419, 0)) {
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/BridgeTest.java:2289]: This test is hard to follow, having this multi-level `if`. I believe this should be extracted into separate test methods, eliminating the conditional for the assertions. Wdyt?

Comment (marcos-iov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeTest.java:2289]: I think we had this same discussion for every PR that add a test here 😬

All these test will be later refactored

File: rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java [hunk 2]
```diff
@@ -404,6 +404,18 @@ public Optional<Integer> getProposedFederationSize() {
...
+     *
+     * <p>
+     * This method checks if a proposed federation exists and, if present,
+     * returns the time at which it was created. The proposed federation is
+     * typically one that is awaiting approval or activation.
+     * </p>
+     *
+     * @return an {@link Optional} containing the {@link Instant} of the proposed federation's creation,
+     *         or an empty {@link Optional} if no proposed federation exists.
+     */
```
[the remark this answers is not in the document]
Comment (apancorb) [reply rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java]: That is a good point, generally they are placed in the interfaces. I have moved all of them to the interface for now.

**Review by nathanieliov — APPROVED** [review]

**Review by marcos-iov — APPROVED** [review]
